roles/nf-core/tasks/pipeline.yml

```diff
@@ -15,13 +15,14 @@
     loop_var: cachedir
 
 - name: nf-core download {{ pipeline }} v{{ release }}
-  command: "nf-core -v
+  command: >
+    nf-core -v
     download {{ pipeline }}
     --compress none
     --outdir {{ sw_path }}/{{ pipeline }}
     --revision {{ release }}
-    --container singularity
-    --singularity-cache-only"
+    --container-system singularity
+    --container-cache-utilisation amend
```

Comment:
I'm not familiar with these parameters but I'll assume they are correct 😄

Reply:
Yes, I had to dig that up too, for the record, this was introduced in these PRs:
- https://github.com/nf-core/tools/pull/2247
- https://github.com/nf-core/tools/pull/2336